packages/driver/cypress/e2e/commands/screenshot.cy.js

```diff
@@ -311,6 +314,9 @@ describe('src/cy/commands/screenshot', () => {
       cy.screenshot('my/file').then(() => {
         expect(Cypress.automation.withArgs('take:screenshot').args[0][1].name).to.equal('my/file')
       })
+
+      // Reset the runnable title to the test name so we can close the span for the test.
+      runnable.title = 'can pass name'
```

Comment:
Looking at the tests, I'm not sure why the `title` is even being changed. I commented the title change out and the test still passed.

Reply:
i removed the test name changes, running tests now to see if they're affected.

Reply:
this seems to not have affected tests.